keps/sig-scheduling/20180409-scheduling-framework.md

```diff
@@ -264,6 +268,31 @@ state, it will either trigger [Un-reserve](#un-reserve) plugins (on failure) or
 
 *Note: This concept used to be referred to as "assume".*
 
+### Pre-Preempt
+
+This is an informational extension point. Plugins will be called with a pod which was unscheduled and eligible to preemption. A plugin may use this data to update internal state or to generate logs/metrics.
+
+### Preempt
```

Comment:
Let's name it more specifically.
As described here, this is not meant to replace the whole preemption logic. It just filters victim candidates.
Maybe `PreemptFilter` or `VictimFilter`

Reply:
NIce.  let's rename it.  How about `VictimsSelection`?